test/role.test.js

```diff
@@ -47,9 +47,6 @@ describe('role model', function() {
     ACL.roleMappingModel = RoleMapping;
     ACL.userModel = User;
     ACL.applicationModel = Application;
-    Role.roleMappingModel = RoleMapping;
-    Role.userModel = User;
-    Role.applicationModel = Application;
```

Comment:
@bajtos with these lines, the original tests implemented a work-around that made my original issue not happen in this module's unit-tests. By setting `Role`'s related models manually, this skipped the process of looking-up loopback's built-in models by prototype and hence prevented the error from happening. If you just remove these lines from the test without implementing my fix, all tests should fail (and hence automatically verify my change)